pybop/_optimisation.py

```diff
@@ -40,27 +40,24 @@ class Optimisation:
     def __init__(
         self,
         cost,
-        x0=None,
         optimiser=None,
         sigma0=None,
         verbose=False,
         physical_viability=True,
         allow_infeasible_solutions=True,
     ):
         self.cost = cost
-        self.x0 = x0
+        self.x0 = cost.x0
```

Comment:
As mentioned above. This keeps it open :)
```suggestion
        self.x0 = x0 or cost.x0
```